### DISCUSSION TITLE
code-first vs product-first

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#14] [General]
# code-first vs product-first

作为程序员，如果你不关心怎么写好代码，那就没有什么要关心的了。就着好代码这个问题聊聊普通程序员最好的状态，也是一段时间以来自己对程序员这个职业的一些理解和思考。

https://www.edony.ink/code-first-vs-product-first/

**Comment by edonyzpc** [comment]
迟到的更新，先跟各位订阅者说一声抱歉，因为换工作等原因迟迟没有更新，后面恢复每周一更。🤓
